### ISSUE TITLE
Modal for when a video cannot be found

### ISSUE THREAD

**Issue by Bboy486** [darklinkpower/PlayniteExtensionsCollection#168]
I am updating the videos for my games and if I select multiple at a time and pick Steam it does not tell me if it does not find a video. I have to look at each individually to see if the video tab (desktop) shows a video. At least with YouTube I can pick from the list. 

Can a pop-up show when a video or the video tab was not added when applying to multiple games?

**Comment by darklinkpower** [comment]
Yeah good idea. I'll add notifications in the future so it doesn't interrupt the download process and the user can know.
